SPHINXsys/src/shared/io_system/in_output.cpp

```diff
@@ -9,6 +9,20 @@
 #include "level_set.h"
 #include "sph_system.h"
 
+namespace
+{
+    /**
+     * @brief Convert any input to string and pad the output with zeros
+     * @todo Use external library for general string formatting, e.g. abseil, fmt library, or std::format
```

Comment:
this is another very good improvement, which can also be discussed in Friday meetings with Prof. Hu